session.go

```diff
@@ -385,7 +395,10 @@ func (s *Session) mux(remoteBegin *performBegin) {
 				s.muxFrameToLink(link, fr.body)
 
 			case *performEnd:
-				s.txFrame(&performEnd{}, nil)
+				if err := s.txFrame(&performEnd{}, nil); err != nil {
+					s.err = err
```

Comment:
what's the consequence of setting the session err ?
when is it checked, and where is it returned?

Reply:
previously, it was only ever set when calling Close() it seems.

Reply:
is failing to send 1 frame a reason to set the error on the session?

Reply:
A fair question.  What happens if we fail to send the frame, are we asked to send again?  The endpoint doesn't care?  It seems dubious to just ignore the error (clearly the frame is important else we wouldn't send it).  Makes me wonder if things like this are part of the flakiness we see with connections getting dropped, but it's just speculation.

Reply:
Digging into this more, `txFrame()` will only return an error if the connection has been closed, and presumably we fail someplace else early on.  So, I think these errors can be safely ignored given the current implementation (the lack of comments explaining this is beyond frustrating).